apps/poker-roll/src/app/startSession.jsx

```diff
@@ -173,23 +173,21 @@ const NewSession = (props) => {
         <div>
             <form
         onSubmit={handleFormSubmit}
-      >
-            <ToggleButtonGroup
-      color="primary"
-      value={gameType}
-      exclusive
-      onChange={handleToggleChange}
-      aria-label="Platform"
-    >
-      <ToggleButton value="cashgame">Cash Game</ToggleButton>
-      <ToggleButton value="tournament">Tournament</ToggleButton>
-      </ToggleButtonGroup>
-      {/* change which stack is show depending on cashgame or tournament is selected */}
-      { gameType === "cashgame" ? <CashGameForm/> : <TournamentForm/>}  
-
-       
-      </form>
-        </div>
+            >
+              <ToggleButtonGroup
+                color="primary"
+                value={gameType}
+                exclusive
+                onChange={handleToggleChange}
+                aria-label="Platform"
+              >
+                <ToggleButton value="cashgame">Cash Game</ToggleButton>
+                <ToggleButton value="tournament">Tournament</ToggleButton>
+              </ToggleButtonGroup>
+                {/* change which stack is show depending on cashgame or tournament is selected */}
+                { gameType === "cashgame" ? <CashGameForm/> : <TournamentForm/>}  
```

Comment:
Noticed I couldn't seem to access these forms anymore, not sure if that was intentional for the time being while you're moving things around.

Reply:
those are intended to be a modal/pop up when you click on start new session from the home screen, that functionality has not been built yet, so they are currently inaccessible